### ISSUE TITLE
Why doesn't yolov5 use "torch.cuda.manual_seed"?

### ISSUE THREAD

**Issue by Hiram1026** [ultralytics/yolov5#7636]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Why doesn't yolov5 use "torch.cuda.manual_seed"?

`import torch.backends.cudnn as cudnn
    random.seed(seed)
    np.random.seed(seed)
    torch.manual_seed(seed)
    cudnn.benchmark, cudnn.deterministic = (False, True) if seed == 0 else (True, False)`

**Comment by glenn-jocher** [comment]
@Hiram1026 YOLOv5 does use this:
https://github.com/ultralytics/yolov5/blob/1a3ecb8b386115fd22129eaf0760157b161efac7/utils/general.py#L186-L194
